lib/kafka/broker.rb

```diff
@@ -30,6 +30,20 @@ def fetch_metadata(**options)
       @connection.send_request(request, response_class)
     end
 
+    def fetch_messages(**options)
```

Comment:
would be nice if you could document things using (for example) yard - otherwise using this gem when it gets bigger and bigger might get harder